Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -86,13 +86,11 @@ static constexpr u32 inet_addr(u8 a, u8 b, u8 c, u8 d)
 
 static int inet_pton(const char* src, unsigned char* dst)
 {
-  int saw_digit, octets;
+  int saw_digit = 0, octets = 0;
```

Comment:
Thoughts on splitting this into two individual declarations? We rarely declare multiple variables like this in other places.
```suggestion
  int saw_digit = 0;
  int octets = 0;
```